static/index.md

```diff
@@ -33,6 +33,25 @@ For more complicated queries the ['Filter a dataset'](filter/) service will be t
 
 [Guide to filtering a dataset](filters/)
 
+### Rate Limiting Rules
+
+Rate limiting has been implemented to contain the number of requests a client can make to our API. This is to protect the service against excessive use and denial of service attacks. 
+
+The following rate limits have been implemented on the API: 
+
+* 120 requests per 10 seconds
+
+* 200 requests per 1 minute
+
+They have been applied to following domains: 
+
+* api.ons.gov.uk
+
+* api.beta.ons.gov.uk
+
+If requests are made that reach these limits the IP address of the client will be blocked for 1 minute. The client will receive a JSON response with a 429 error code, the retry-after header shows the seconds remaining until a retry can be made.  
```

Comment:
The client will receive a JSON response with a 429 error code. The Retry-After header shows the seconds remaining until a retry can be made.